app/controllers/organizations_controller.rb

```diff
@@ -24,4 +24,12 @@ def show
         @profile_image_url = @organization.profile_image.url
     end
   end
+
+  def edit
+    @organization = Organization.find(params[:id])
+    @profile_image_url = nil
```

Comment:
@ethanlee16 + @SaladRaider + @clairelin135 

When editing profile_mage_url is this suppose to not be nil? (where should we be sourcing the image?)